app/models/comment.rb

```diff
@@ -2,4 +2,5 @@ class Comment < ApplicationRecord
   include Visible
 
   belongs_to :article
+  has_one :user, through: :article
```

Comment:
This should be belongs_to right? Any reason using has_one?

Reply:
At first I saw this:
https://guides.rubyonrails.org/association_basics.html
I now realise the above is shit because I found this:
https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/has_one
Comment class is the one with the foreign. Will change to belongs_to.

Reply:
Done